aws/resource_aws_eip.go

```diff
@@ -108,6 +108,12 @@ func resourceAwsEip() *schema.Resource {
 				Computed: true,
 			},
 
+			"network_border_group": {
```

Comment:
`ForceNew: true` as it doesn't seem like the network border group can be updated after creation.